redisinsight/ui/src/pages/browser/components/key-details-add-items/add-stream-entity/StreamEntryFields/StreamEntryFields.tsx

```diff
@@ -77,10 +145,13 @@ const AddStreamEntity = (props: Props) => {
                 <EuiIcon type="iInCircle" style={{ cursor: 'pointer' }} />
               </EuiToolTip>
             )}
-            data-testid={config.entryId.name}
+            isInvalid={!!entryIdError}
+            autoComplete="off"
+            data-testid={config.entryId.id}
           />
         </EuiFormRow>
-        <span className={styles.timestampText}>Timestamp - Sequence Number or *</span>
+        {!showEntryError && <span className={styles.timestampText}>Timestamp - Sequence Number or *</span>}
+        {showEntryError && <span className={styles.error}>{entryIdError}</span>}
```

Comment:
Can we replace it with showEntryError ? <span className={styles.error}>{entryIdError}</span> ? <span className={styles.timestampText}>Timestamp - Sequence Number or *</span>